src/material/core/tokens/m2/mdc/_progress-spinner.scss

```diff
@@ -0,0 +1,56 @@
+@use 'sass:map';
```

Comment:
This file corresponds to the tokens for one of MDC's components (specifically mdc-circular-progress) so it should be named to reflect that `m2/mdc/_circular-progress.scss`. I was imagining that if we need to create our own custom tokens for our progress spinner they will live in `m2/mat/_progress-spinner.scss`